Source/WebCore/platform/graphics/gstreamer/eme/CDMOpenCDM.cpp

```diff
@@ -99,9 +98,9 @@ std::unique_ptr<CDMPrivate> CDMFactoryOpenCDM::createCDM(const String& keySystem
     return std::unique_ptr<CDMPrivate>(new CDMPrivateOpenCDM(keySystem));
 }
 
-bool CDMFactoryOpenCDM::supportsKeySystem(const String& keySystem)
+bool CDMFactoryOpenCDM::supportsKeySystem(const String& /* keySystem */)
 {
-    return GStreamerEMEUtilities::isPlayReadyKeySystem(keySystem) || GStreamerEMEUtilities::isWidevineKeySystem(keySystem);
+    return true;
```

Comment:
See last commit. Although I still think that, also prior to this change, the "pericodelospalotes.keysystem.com" would not lead to a DRM session since another call is made to IsTypeSupported that will return later on when the mime types are available. 
Now the call to IsTypeSupported from an OpenCDM level is done twice.

Reply:
We discussed this change a while ago, we already do key system checks in the CDMPrivate's supportsConfiguration member call (the thing that is supposed to check supported mime / init data types). By returning false early here in the factory, all we avoid is creating a CDMPrivate, which is a tiny object that isn't worth sweating about. The m_openCdm member will now have to be added the CDMFactoryOpenCDM so it's still getting created. The only alternative I see to avoid doing a cross-process request for this is to hardcode what keysystems we support in opencdm into webkit, this is ugly, but practical. We don't add new keysystem support very often..

Anyway, that was just a dump of my recollection of the last conversation we had about this small point :-)